public/index.html

```diff
@@ -0,0 +1,370 @@
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/icon-styles.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <!-- menu -->
+    <nav class="nav top-nav">
+      <ul class="flex-container flex-navbar">
+        <li class="flex-nav-1"><a href="#home">Home.</a></li>
+        <li class="flex-nav-2"><a href="#">Services.</a></li>
+        <li class="flex-nav-3"><a href="#">Portfolio.</a></li>
+        <li class="flex-nav-4"><a href="#">About.</a></li>
+        <li class="flex-nav-5"><a href="#">Team.</a></li>
+        <li class="flex-nav-6"><a href="#">Contact.</a></li>
+      </ul>
+    </nav>
+    <!-- header -->
+    <header class="flex-container header-notch">
+      <section class="headertext">
```

Comment:
`section`s contain `header`s; not the other way around. If you _need_ another element for styling purposes (which you don't here) a `div` would be more appropriate.